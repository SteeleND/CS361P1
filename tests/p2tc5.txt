b
a

aeb a0a a1a beb b0b b1b
0
000
011
e
10111
