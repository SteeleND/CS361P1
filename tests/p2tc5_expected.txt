Q = { [a, b] }
Sigma = { 0 1 }
delta =
                   0         1
    [a, b]    [a, b]    [a, b]
q0 = [a, b]
F = { [a, b] }

yes
yes
yes
yes
yes
